packages/contents/src/drivefs.ts

```diff
@@ -41,13 +43,42 @@ export type TDriveMethod =
   | 'put';
 
 /**
- * Interface of a request on the /api/drive endpoint
+ * Type of the data argument for the drive request, based on the request name
  */
-export interface IDriveRequest {
```

Comment:
Maybe we could document some of these refactoring changes in https://jupyterlite.readthedocs.io/en/latest/migration.html.

Not expecting them to be use extensively downstream, but still useful to have them documented somewhere.

Reply:
Done!